crypto-project-app/src/App.tsx

```diff
@@ -1,33 +1,38 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+
 import './App.css'
+import Header from './Header'
+import CoinDetails from './CoinDetails'
+import Items from './Items'
+import { useState } from 'react'
+
+
+// const Item =(props)=>(
+// <div className='item flex'>
+//   <div className='name'>{`${props.content.name}`} </div>
+//   <div className='price'>{`${props.content.price}`} </div>
+//   <div className='marketCup'> {`${props.content.marketCup}`} </div> 
+//   <button onClick={()=>changeId(props.content.id)}>Check</button>
+// </div>
+// )
+
+// const checkCoin=(id)=>{
+//   console.log(id)
+// }
+const App =(props:Object)=> {
```

Comment:
Główna aplikacja raczej nie przyjmuje props. Zostaw po prostu `() => { ...` 
Component `CoinDetails` oraz `Items` dobrze będzie umieścić w komponencie mądrym np. Crypto i tam dodaj dane oraz zarządzanie stanem poprzez useState`.
Ułatwi ci to potem zadanie z routingiem, gdyż w komponent App.tsx jest naszym główym komponentem. A plik main.tsx przeważnie zostawiamy jak jak był, czy tylko uruchamia apkę.